sycl-jit/jit-compiler/CMakeLists.txt

```diff
@@ -1,3 +1,4 @@
+find_package(LLVMGenXIntrinsics QUIET)
```

Comment:
Can you elaborate how this works/why this is needed? Looks like we relied on the `LLVMGenXIntrinsics` target being handled by `SYCLLowerIR`, and we don't use any of the variables set by the package directly AFAICT.

Reply:
Fixed in latest commit, thx